docs/deployment/deploy_gardenlet_manually.md

```diff
@@ -0,0 +1,402 @@
+# Deploy a Gardenlet Manually
+
+Deploying a gardenlet into a Kubernetes cluster manually and registering this cluster as a seed cluster automatically is only necessary in special deployment scenarios, for example, to manage clusters in a fenced environment.
+
+Once you’ve deployed the initial gardenlet manually, you can deploy new gardenlets automatically using the initial gardenlet as a template.
+
+More information: [Automatic Deployment of Gardenlets](deploy_gardenlet_automatically.md).
+
+## Prerequisites
+
+### Kubernetes cluster that should be registered as a seed cluster
+    
+- Verify that the cluster has a [supported Kubernetes version](../usage/supported_k8s_versions.md).
+- Determine the nodes, pods, and services CIDR of the cluster. You need to configure this information in the `Seed` configuration. Gardener uses this information to check that the shoot cluster isn’t created with overlapping CIDR ranges.

```

Comment:
```suggestion
- Determine the nodes, pods, and services CIDR of the cluster. 
You need to configure this information in the `Seed` configuration. 
Gardener uses this information to check that the shoot cluster isn’t created with overlapping CIDR ranges.
```
I would use newlines after each sentence. Otherwise the sentence is hard to maintain when displayed as source code on laptops.

Reply:
Same for all other occasions.

Reply:
I inserted more newlines everywhere as good as I could